package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "pay-js-metrics",
-  "version": "1.0.0",
+  "name": "@govuk-pay/pay-js-metrics",
+  "version": "1.0.1",
   "description": "GOV.UK Pay Express middleware for Prometheus metrics instrumentation",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "files": ["dist"],
```

Comment:
Is `npm install` supposed to create a dist folder in the root directory?

Reply:
`npm run build` does that.